Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -187,6 +187,36 @@
     damageCoefficient: 0.4
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitSecurity
+    
+#Security Patrol Hardsuit
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitSecuritypatrol
+  name: security patrol hardsuit
+  description: A special suit that protects from the danger of space, employed by security patrol officers.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Hardsuits/security-patrol.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Hardsuits/security-patrol.rsi
+  - type: PressureProtection
+    highPressureMultiplier: 0.525
+    lowPressureMultiplier: 10000
+  - type: ClothingSpeedModifier
+    walkModifier: 0.85
+    sprintModifier: 0.85
```

Comment:
sprintModifier: 0.80 ~
To be a middle point from sec to brig suit

Reply:
Dig it. Will send another commit with the correction.